docs/tutorial/force_calibration.rst

```diff
@@ -0,0 +1,95 @@
+Force calibration
+=================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+Force calibration is used to convert the raw voltages that Bluelake records to actual forces.
+In this tutorial, we will be redoing a calibration that we already performed in Bluelake, to provide an example of using the force calibration module::
+
+    f = lk.File("calibration.h5")
+
+In this tutorial, we want to reproduce calibration item 1.
+We grab the chunk of data that was used for the calibration last time::
+
+    start = f.force1x.calibration[1]["Start time (ns)"]
+    stop = f.force1x.calibration[1]["Stop time (ns)"]
+    force_slice = f.force1x[start:stop]
+
+To be able to calibrate our forces, we first convert back our data to raw voltages.
+We can do this using the previous calibration performed in Bluelake (in this case, the first one in the file)::
+
+    volts = force_slice.data / f.force1x.calibration[0]["Rf (pN/V)"]
+
+Force calibration models are fit to power spectra. To compute a power spectrum from our data::
+
+    power_spectrum = lk.calculate_power_spectrum(volts, sample_rate=force_slice.sample_rate)
```

Comment:
This function returns an instance of the `PowerSpectrum` class, but that class is in the `detail` namespace and not found in the API reference. So users don't know what they can do with this result (apart from what's described in this tutorial). I would suggest moving that class out of `detail` and adding it to the API reference.

Notes on the `PowerSpectrum` API itself: 
* The uppercase `.P` as an attribute goes against Python conventions. I'm not sure how consistent we are with this. There is an argument to be made about keeping it close to scientific notation. What I don't like is that this messes with most Python tools. E.g. even simple syntax highlighting ends up marking that as a type instead of a variable. I would sooner go with `.power` and `.frequency` for those attributes. But that's my opinion. It might be good if a few of us had a chat about these conventions and maybe write down a quick style guide to define what we want in situations like these.
* I see that `PowerSpectrum` still has `as_dict()`. You can drop that.
* The docstring for `calculate_power_spectrum()` doesn't mention that it returns a `PowerSpectrum`.

Reply:
> The uppercase .P as an attribute goes against Python conventions. I'm not sure how consistent we are with this. There is an argument to be made about keeping it close to scientific notation. What I don't like is that this messes with most Python tools. E.g. even simple syntax highlighting ends up marking that as a type instead of a variable. I would sooner go with .power and .frequency for those attributes. But that's my opinion. It might be good if a few of us had a chat about these conventions and maybe write down a quick style guide to define what we want in situations like these.
I also feel `.power` and `.frequency` is better.
> I see that PowerSpectrum still has as_dict(). You can drop that.
Dropped.
> The docstring for calculate_power_spectrum() doesn't mention that it returns a PowerSpectrum
Amended!